MapboxVisionAR/Render/ARScene/ConcreteNodes/ARCameraNode/ARCameraNode.swift

```diff
@@ -0,0 +1,70 @@
+import simd
+
+class ARCameraNode: ARNode {
+    // MARK: - Properties
+
+    /// Type of node. Always returns `rootNode`.
+    private(set) var nodeType: ARNodeType
+    /// Underlying AR entity.
+    var entity: AREntity?
+    /// Describes position of the node in the node hierarchy.
+    var relations: NodeRelations
+    /// Describes transformation between coordinate systems.
+    var geometry: NodeGeometry
+
+    var needsUpdateProjection = true
+    var nearClipPlane: Float = 0.01 {
+        didSet {
+            setNeedsUpdateProjection()
+        }
+    }
+    var farClipPlane: Float = 1000 {
+        didSet {
+            setNeedsUpdateProjection()
+        }
+    }
+    var fovRadians = degreesToRadians(60) {
+        didSet {
+            setNeedsUpdateProjection()
+        }
+    }
+    var aspectRatio = Float(4.0 / 3.0) {
+        didSet {
+            setNeedsUpdateProjection()
+        }
+    }
+
+    // MARK: - Private properties
+
+    private(set) var cachedProjectionMatrix = float4x4()
```

Comment:
The property has public getter but placed in private section

Reply:
Fixed.